README.md

```diff
@@ -18,4 +25,17 @@ The top menu has a "Region" button which leads the user into a region comparison
 
 The intent of this app is for users to be able to explore both global trends as well as region by region comparisons in an iterative manner.    
 
+## Get Involved
+
+We are always looking for feedback and contributors! 
+
+In second milestone, the main comparison functionality of the "Region" tab was incorporated into landing page to limit the scope of having to build two tabs. Potential functionality to add for the future include:
+
+- ability to select countries on the map by clicking as a more intuitive alternative than typing in the country field
+- exploring ways to speed up the page load time (loading widget currently implemented)
+
+The next feature we are hoping to build is a second tab with all the features outlined in the app sketch image directly above. You can run the app locally and contribute by:
```

Comment:
I think you may need a separate line between the first 2 sentences to set them apart. Here looks like they are joined together

Reply:
I realized the first sentence makes no sense. It's from an earlier version. I removed completely.